test/modules/Modules.Test/TestResultCoordinator/Reports/DirectMethod/DirectMethodLongHaulReportGeneratorTest.cs

```diff
@@ -330,10 +330,10 @@ public async Task TestCreateReportAsyncWithEmptyResults()
 
             var report = (DirectMethodLongHaulReport)await reportGenerator.CreateReportAsync();
 
-            Assert.Equal(0UL, report.ReceiverSuccesses.Expect<ArgumentException>(() => throw new ArgumentException("impossible")));
-            Assert.Equal(0UL, report.SenderSuccesses);
-            Assert.Equal(0UL, report.StatusCodeZero);
-            Assert.Equal(0UL, report.Unknown);
+            Assert.Equal(0L, report.ReceiverSuccesses.Expect<ArgumentException>(() => throw new ArgumentException("impossible")));
+            Assert.Equal(0L, report.SenderSuccesses);
+            Assert.Equal(0L, report.StatusCodeZero);
+            Assert.Equal(0L, report.Other.Sum(x => x.Value));
             Assert.True(report.IsPassed);
```

Comment:
From the look there is no test case that verify the sorting of `statusCode` key here? From the PR, it looks like it should work. You can also add a new test case here too. I'll leave it up to you to decide.